core/inject.js

```diff
@@ -379,9 +379,14 @@ Blockly.init_ = function(mainWorkspace) {
   }
 
   if (options.moveOptions && options.moveOptions.scrollbars) {
+    var horizontalScroll = options.moveOptions.scrollbars === true ||
+        options.moveOptions.scrollbars.horizontal;
```

Comment:
if `scrollbars = false` horziontalScroll will be equal to undefined. Which is okay the way we're using it and happen to not lie about its type (ScrollbarPair param type is `boolean=`), however I think it's been using as a boolean in this context.

Prefer if you did `!!options.moveOptions.scrollbars.horizontal` here and below.

Reply:
sgtm